BBallStatsTrack/src/com/example/bballstatstrack/models/gameevents/TurnoverEvent.java

```diff
@@ -0,0 +1,69 @@
+package com.example.bballstatstrack.models.gameevents;
+
+import com.example.bballstatstrack.models.Player;
+import com.example.bballstatstrack.models.Team;
+import com.example.bballstatstrack.models.gameevents.foulevents.OffensiveFoulEvent;
+
+public class TurnoverEvent extends GameEvent
+{
+    public static final String TURNOVER_TYPE = "turnoverType";
+
+    private TurnoverType mTurnoverType;
+
+    public TurnoverEvent( TurnoverType type, Player player, Team team )
+    {
+        super( EventType.TURNOVER, player, team );
+        mTurnoverType = type;
+    }
+
+    @Override
+    public void append( GameEvent appendedEvent )
+    {
+        if( mAppended != null )
+        {
+            mAppended.append( appendedEvent );
+            return;
+        }
+        if( appendedEvent instanceof StealEvent && mTurnoverType == TurnoverType.STEAL )
+        {
+            mAppended = appendedEvent;
+            return;
+        }
+        else if( appendedEvent instanceof OffensiveFoulEvent && mTurnoverType == TurnoverType.OFFENSIVE_FOUL )
+        {
+            FoulType foulType = ( ( FoulEvent ) appendedEvent ).getFoulType();
+            if( foulType == FoulType.OFFENSIVE )
+            {
+                mAppended = appendedEvent;
+                return;
+            }
+        }
+        super.append( appendedEvent );
```

Comment:
can be removed since GameEvent.append(GameEvent appendedEvent) does nothing po..